qemu/tests/cfg/live_snapshot.cfg

```diff
@@ -85,10 +85,9 @@
                 i386, i686:
                     install_path = "C:\Program Files\JAM Software\HeavyLoad"
             Linux:
-                stress_test = autotest_control
-                test_control_file = stress.control
+                stress_test = linux_stress
                 # Use a low stress to make sure guest can response during stress
-                control_args = "--cpu 4 --io 4 --vm 2 --vm-bytes 256M"
+                stress_args = "--cpu 1 --io 1 --vm 1 --vm-bytes 128M"
```

Comment:
Why do you choose these values?

Reply:
@sitoliu Because the comment in the above line 89 `# Use a low stress to make sure guest can response during stress`